monument/test/results.toml

```diff
@@ -737,6 +738,118 @@ len    Y   C |  music  | avg score | calling
 -----------------------|----------|--------------------------------------------------------------------------------------|-----------|-----------
  len    D   L   S   Y  |    PH    |  music       4-bell runs      5-bell runs    6-bell runs   7-bell runs   8-bell runs | avg score | calling
 '''
+"test/cases/complib-music/caters.toml" = '''
+len |  music     *56789   *65789       4-bell runs    | avg score | calling
+----|-------------------------------------------------|-----------|-----------
+144 |   80.00 :     4        0       22 (  11f   11b) |  0.555556 | 
+288 |   83.00 :     4        0       30 (  15f   15b) |  0.259722 | sMsHMH
+288 |   83.00 :     4        0       30 (  15f   15b) |  0.263194 | MHMH
+288 |   85.00 :     4        0       32 (  16f   16b) |  0.263194 | sMsHsMsH
+288 |   85.00 :     4        0       32 (  16f   16b) |  0.266667 | MHsMsH
+288 |  105.00 :     8        0       34 (  17f   17b) |  0.348611 | sHsH
+432 |  115.00 :     8        0       44 (  22f   22b) |  0.244907 | sHsMsMsH
+432 |  126.00 :     8        0       54 (  27f   27b) |  0.272685 | HWsWsH
+432 |  126.00 :    12        0       44 (  22f   22b) |  0.279167 | HHH
+432 |  127.00 :     8        0       54 (  27f   27b) |  0.272685 | sHsWsWsH
+----|-------------------------------------------------|-----------|-----------
+len |  music     *56789   *65789       4-bell runs    | avg score | calling
+'''
+"test/cases/complib-music/cinques.toml" = '''
+len |  music     *567890E   *657890E       4-bell runs    | avg score | calling
+----|-----------------------------------------------------|-----------|-----------
+440 |  107.00 :      4          0        30 (  15f   15b) |  0.226818 | WHWH
```

Comment:
I wonder if we could make the music displayed by Monument also match exactly the 'header line' of Complib?  This is a little tricky because CompLib hides music which doesn't appear, so it's worth finding some musical examples for each stage to see what CompLib displays here...

I think many of these are very close actually.  Often the major difference is to give 56s/65s/crus more succinct names, which can be done by adding e.g. `name = "56s"`.  This is already done for 65s/87s/09s/etc.:
https://github.com/kneasle/ringing/blob/master/monument/cli/src/default-music-major.toml#L25